pkgs/by-name/ws/wstunnel-rust/package.nix

```diff
@@ -0,0 +1,32 @@
+{ fetchFromGitHub
+, rustPlatform
+, lib
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wstunnel-rust";
+  version = "9.6.1";
+
+  src = fetchFromGitHub {
+    owner = "erebe";
+    repo  = "wstunnel";
+    rev = "v${version}";
+    sha256 = "sha256-+tNQf53KUNIz9RFVTHqgXEKLxlXiQKKPZtXqD3C/t7I=";
+  };
+
+  cargoHash = "sha256-NER7NIWUXyWFUGcdZ5HLFaKaO9FNLPlG7jql08karS4=";
+
+  checkFlags = [
+    # make use of network connection
+    "--skip=tcp::tests::test_proxy_connection"
+  ];
+
+  meta = with lib; {
+    description = "Wstunnel uses the websocket protocol which is compatible with http in order to bypass firewalls and proxies.";
+    mainProgram = "wstunnel";
+    homepage    = "https://github.com/erebe/wstunnel";
+    license     = with licenses; [ bsd3Clear ];
```

Comment:
Doesn't seem to be `bsd3Clear`, not sure which variant of bsd license either.

Reply:
How to include the license here? It does sounds like a BSD

Reply:
There is a concerning line: `All rights reserved`, better ask author for clarification.

Reply:
I think this is from https://spdx.org/licenses/BSD-3-Clause-Clear.html ?

Reply:
But missing the `NO EXPRESS OR IMPLIED LICENSES TO ANY PARTY'S PATENT RIGHTS ARE GRANTED BY THIS LICENSE` wording about patent.

Reply:
Open a discussion at the project: https://github.com/erebe/wstunnel/discussions/293

Reply:
mark as unfree until we have update from the author

Reply:
Author makes clear that it is BSD3